environment/terraform.tfvars.json

```diff
@@ -16,7 +16,8 @@
       "mock_emails": false,
       "symfony_env": "prod",
       "db_subnet_group": "rds-private-subnets-prod-vpc",
-      "ec_subnet_group": "ec-pvt-subnets-prod-vpc"
+      "ec_subnet_group": "ec-pvt-subnets-prod-vpc",
+      "sirius_api_account": "649098267436"
```

Comment:
I would've expected a separate account for the production API gateway. Is prod and preprod shared, or has it just not been set up yet @jamesrwarren?

Reply:
Hasn't been set up as far as I know. Will ask Tom today as he was wokring on it and we need to push the deploy out to pre and prod.

Reply:
From Slack: Other environments are still in the process of being set up so we'll review these account IDs once that work's complete.